external/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.digitalpebble</groupId>
+        <artifactId>storm-crawler</artifactId>
+        <version>0.4-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>storm-crawler-external</artifactId>
+    <packaging>jar</packaging>
+
+    <name>storm-crawler-external</name>
+    <url>https://github.com/DigitalPebble/storm-crawler/external</url>
+    <description>Storm-Crawler Java API with external dependencies.</description>
+
+    <properties>
+        <!-- dependency versions -->
+        <elasticsearch.version>1.3.2</elasticsearch.version>
+        <metrics-core.version>3.0.0</metrics-core.version>
+        <librato-java.version>0.1.2</librato-java.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <!-- no test jar for storm-crawler-external -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-jar-plugin</artifactId>
+                        <version>2.5</version>
+                        <executions>
+                            <execution>
+                                <id>attach-test</id>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
```

Comment:
I would personally delete the whole `<profiles>` block for external since we don't use it right now